### PR TITLE
Upgrade SR-IOV provider nodes image to k8s-1.22

### PR DESCRIPTION
**Pull request by ormergi**
**What this PR does / why we need it:**
Following https://github.com/kubernetes-sigs/kind/issues/2318, https://github.com/kubernetes-sigs/kind/pull/2321 and https://github.com/kubernetes-sigs/kind/pull/2465

The new KinD k8s-1.22 nodes image includes all the necessary changes that allows us to finally stop using `PodPresets`, upgrade k8s above 1.19 and still support Kubevirt Live Migration!

**Thanks to @qinqon for his contribution to KinD upstream that allowed this to happen :crown:**

**Note to reviewers:**
PodPreset related files are not delete yet, it seems that the GPU providers uses it.

Signed-off-by: Or Mergi <ormergi@redhat.com>

### CODE REVIEW

File: cluster-up/cluster/kind-1.22-sriov/provider.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ SRIOV_TESTS_NS="${SRIOV_TESTS_NS:-kubevirt-test-default1}"
 
 function set_kind_params() {
     export KIND_VERSION="${KIND_VERSION:-0.11.1}"
-    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-quay.io/kubevirtci/kindest_node:v1.19.11@sha256:cbecc517bfad65e368cd7975d1e8a4f558d91160c051d0b1d10ff81488f5fb06}"
+    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.22.2@sha256:f638a08c1f68fe2a99e724ace6df233a546eaf6713019a0b310130a4f91ebe7f}"
```
Comment (qinqon) [inline cluster-up/cluster/kind-1.22-sriov/provider.sh:22]: We have to point to quay, can you work with @fgimenez to push it and then we change this PR ?

Comment (qinqon) [inline cluster-up/cluster/kind-1.22-sriov/provider.sh:22]: We have to point to quay, can you work with @fgimenez to push it and then we change this PR ?

Comment (ormergi) [reply cluster-up/cluster/kind-1.22-sriov/provider.sh]: @fgimenez Wait does mirroring to quay.io/kubevirt is done automaticly now? :scream:

File: cluster-up/cluster/kind-1.22-sriov/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+export KUBEVIRT_PROVIDER=kind-1.22-sriov
+make cluster-down
+```
+
+This destroys the whole cluster, and moves the SR-IOV nics to the root network namespace.
+
+## Setting a custom kind version
+
+In order to use a custom kind image / kind version, export `KIND_NODE_IMAGE`, `KIND_VERSION`, `KUBECTL_PATH` before
+running cluster-up. For example in order to use kind 0.9.0 (which is based on k8s-1.19.1) use:
```
Comment (qinqon) [inline cluster-up/cluster/kind-1.22-sriov/README.md:59]: There are some 1.19 eferences.

Comment (qinqon) [reply cluster-up/cluster/kind-1.22-sriov/README.md:59]: fair enough

**Pull request merged**